---
renderer:
  layer_z_step: 5.5  # unit separation between layers
  collapse_layers: false  # whether or not to apply offsets to each of the layers
  layers:
    2:
      z_offset_scale: 2.0
      visualize: true
      nodes: {scale: 0.40, color: {type: LabelColorAdapter}, alpha: 0.8, use_sphere: false}
      text: {draw: true, collapse: true, adapter: {type: LabelTextAdapter}, height: 0.5, scale: 0.45}
      bounding_boxes: {draw: true, collapse: true, scale: 0.05, edge_scale: 0.05, alpha: 0.9, edge_break_ratio: 0.5}
      edges: {interlayer_use_source: true, interlayer_scale: 0.08, interlayer_alpha: 0.9, interlayer_use_color: true}
    3:
      z_offset_scale: 3.0
      visualize: true
      nodes: {scale: 0.2, color: {type: ParentColorAdapter, colormap: {palette: colorbrewer}}, alpha: 0.9, use_sphere: true}
      edges:
        scale: 0.01
        alpha: 0.5
        use_color: false
        interlayer_use_source: false
        interlayer_scale: 0.08
        interlayer_alpha: 0.4
        interlayer_use_color: true
        interlayer_insertion_skip: 0
    4:
      z_offset_scale: 4.2
      visualize: true
      nodes: {scale: 0.6, color: {type: IdColorAdapter, colormap: {palette: colorbrewer}}, alpha: 0.8, use_sphere: false}
      text: {draw: true, height: 1.25, scale: 1.0}
      edges:
        scale: 0.1
        alpha: 0.2
        use_color: false
        interlayer_use_source: true
        interlayer_scale: 0.08
        interlayer_alpha: 0.4
        interlayer_use_color: true
        interlayer_insertion_skip: 0
  partitions:
    2:
      z_offset_scale: 0.0
      visualize: true
      nodes: {scale: 0.15, alpha: 0.9, use_sphere: false, color: {type: PartitionColorAdapter}}
      edges: {scale: 0.05, alpha: 0.9, draw_interlayer: false}
      text: {draw_layer: true, height: 0.9, scale: 0.8}
    3:
      z_offset_scale: 3.0
      visualize: true
      nodes: {scale: 0.2, color: {type: LabelColorAdapter}, alpha: 0.9, use_sphere: true}
      boundaries: {draw: true, collapse: false, wireframe_scale: 0.1, use_node_color: true, alpha: 1.0}
      edges:
        scale: 0.01
        alpha: 0.5
        use_color: false
        interlayer_use_source: false
        interlayer_scale: 0.08
        interlayer_alpha: 0.4
        interlayer_use_color: true
        interlayer_insertion_skip: 0
